content/hi/docs/reference/glossary/kubeadm.md

```diff
@@ -0,0 +1,18 @@
+---
+title: क्यूबएडीएम  (Kubeadm)
```

Comment:
Small nit. 
```suggestion
title: क्यूबएडीएम (Kubeadm)
```

Reply:
The above changes were suggested by someone, thank you for the fix

<img width="832" alt="Screenshot 2022-11-21 at 10 38 35 AM" src="https://user-images.githubusercontent.com/51878265/202970524-90d30a27-81bb-4f46-9223-d8ed04d0f1db.png">